### DISCUSSION TITLE
Feature Requiest: View Logs for running step

### DISCUSSION THREAD

**Discussion by bitdeft** [community/community#35363] [Actions]
An issue was raised for this; details and further information can be seen here: 

https://github.com/actions/runner/issues/886

@ethomson said that such feature requests should be raised in the discussion forum here. I am working with build debugging actively and can add a +1 to making this feature a priority, especially with steps that take a long time to complete, and provide vital information during their execution for debugging purposes.

**Comment by bitdeft** [comment]
Actually I searched and found this brought up a few times already in discussion. Please close this out as a duplicate, or perhaps, use as another tally on the board for getting this feature bumped up in priority? Thanks

**Reply by venkatamutyala** [reply]
Is there a work around? Anyone to have an action that streams like by line to slack or something else? Or maybe ships to elastic search?

**Comment by dongho-jung** [comment]
It's been almost two years after the first issue was opened, and there has been no feedback at all...

**Comment by axel-h** [comment]
Related discussion: #44250

**Comment by mcarans** [comment]
Potentially related documentation issue: https://github.com/github/docs/issues/25041
